internal/store/alert_configuration.go

```diff
@@ -40,3 +45,16 @@ func (s *Store) AlertConfigurations(projectID string, opts *atlas.ListOptions) (
 		return nil, fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+// CreateAlertConfiguration encapsulate the logic to manage different cloud providers
+func (s *Store) CreateAlertConfiguration(alert *atlas.AlertConfiguration) (*atlas.AlertConfiguration, error) {
```

Comment:
[nit] instead of alert let's do something like, `ac`(alert config, go likes short names), or `config` or `alertConfig`, will add alerts later so trying to to confuse this with that